apps/transport/lib/transport/import_data.ex

```diff
@@ -442,11 +442,25 @@ defmodule Transport.ImportData do
     end
   end
 
-  def is_ods_title?(%{"title" => title})
+  @doc """
+  iex> is_ods_resource?(%{"format" => "json", "title" => "GTFS (json)", "harvest" => %{"uri" => "https://data.angers.fr/api/explore/v2.1/catalog/datasets/angers-loire-metropole-horaires-reseau-irigo-gtfs-rt/exports/json"}})
+  true
+  iex> is_ods_resource?(%{"format" => "GTFS", "title" => "GTFS été"})
+  false
+  iex> is_ods_resource?(%{"format" => "csv", "title" => "Export au format CSV"})
+  true
+  """
+  def is_ods_resource?(%{"title" => title})
       when title in ["Export au format CSV", "Export au format JSON"],
       do: true
 
-  def is_ods_title?(_), do: false
+  def is_ods_resource?(%{"harvest" => %{"uri" => uri}}) do
+    # Possible URL:
+    # https://data.angers.fr/api/explore/v2.1/catalog/datasets/angers-loire-metropole-horaires-reseau-irigo-gtfs-rt/exports/json
+    String.match?(uri, ~r{/api/explore/v(\d+\.\d+)/catalog/datasets/.*/exports/(json|csv)$})
```

Comment:
Je propose de passer en case-insensitive pour être plus flexible sur ce coup là, on ne sait jamais:

```suggestion
    String.match?(uri, ~r{/api/explore/v(\d+\.\d+)/catalog/datasets/.*/exports/(json|csv)$}i)
```